### PR TITLE
Propagate executor paymaster error

### PR DESCRIPTION
**Pull request by tarrencev**

### CODE REVIEW

**Comment by JunichiSugiura** [comment]
Screenshot 2024-09-05 at 14 38 56

**Comment by JunichiSugiura** [comment]
WIP

- DeploymentRequired
  - Check if funding is required
    - Yes
      - Render `<Funding />`
      - wait for tx
      - Change CTA to deploy
  - Deploy

File: packages/keychain/src/components/Funding.tsx [hunk 1]
```diff
@@ -61,100 +49,88 @@ export function Funding(innerProps: FundingInnerProps) {
...
+      ];
       const res = await extAccount.execute(calls);
       await extAccount.waitForTransaction(res.transaction_hash, {
         retryInterval: 1000,
+        successStates: [
+          TransactionExecutionStatus.SUCCEEDED,
+          TransactionFinalityStatus.ACCEPTED_ON_L2,
+        ],
       });
+      onComplete(res.transaction_hash);
```
[the remark this answers is not in the document]
Comment (JunichiSugiura) [reply packages/keychain/src/components/Funding.tsx:126]: We should simply check the balance then

File: packages/keychain/src/components/ErrorAlert.tsx [hunk 2]
```diff
@@ -71,3 +140,218 @@ export function ErrorAlert({
...
+                    <HStack
+                      key={key}
+                      w="full"
+                      justifyContent="space-between"
+                      fontSize="xs"
+                    >
+                      <Text color="darkGray.400">{key}</Text>
+                      {key === "Address" || key === "Class" ? (
+                        <Link
+                          href={`https://starkscan.co/${
```
Comment (broody) [inline packages/keychain/src/components/ErrorAlert.tsx:330]: Links always point to mainnet starkscan currently

**Pull request merged**